sdk/formrecognizer/ai-form-recognizer/src/formTrainingClient.ts

```diff
@@ -515,10 +515,16 @@ export class FormTrainingClient {
   }
 
   /**
-   * Generates authorization for copying a custom model into this Azure Form Recognizer resource.
+   * Generate an authorization for copying a custom model into this Azure Form Recognizer resource.
+   *
+   * This method should be called on a client that is authenticated using the target resource (where the
+   * model will be copied to) credentials, and the output can be passed as the `target` parameter to the
+   * `beginCopyModel` method of a source client.
+   *
+   * The required `resourceId` and `resourceRegion` of an Azure Form Recognizer resource can be found in the Azure Portal, under the "Properties" section of your Cognitive Services resource (under the "Resource Management" header).
```

Comment:
general guidance is not to be too specific of where to find things in the portal since this information changes quite frequently